components/select/Select.js

```diff
@@ -0,0 +1,77 @@
+import React, { PureComponent } from 'react';
+import ReactSelect from 'react-select';
+import PropTypes from 'prop-types';
+import { IconChevronDownSmallOutline } from '@teamleader/ui-icons';
+import { Button } from '../button';
+import theme from './theme.css';
+
+class Select extends PureComponent {
+  state = {
+    selectedOptions: [],
+  };
+
+  getDropDownIndicator = () => ({ isDisabled }) => (
+    <Button
+      className={theme['dropdown-indicator']}
+      disabled={isDisabled}
+      icon={<IconChevronDownSmallOutline />}
+      size={this.props.size}
+    />
+  );
+
+  getStyles = (size) => ({
+    control: (base, { isDisabled, isFocused }) => ({
+      ...base,
+      backgroundColor: isDisabled ? '#e4e4e6' : '#fff',
+      '&:hover': {
+        borderColor: '#82828c'
+      },
+      borderColor: isFocused ? '#82828c' : '#c0c0c4',
+      boxShadow: isFocused ? '0 0 0 1px #82828c' : 'none',
+      minHeight: size === 'small' ? '30px' : size === 'medium' ? '36px' : '48px',
+    }),
+    indicatorSeparator: (base) => ({
+      ...base,
+      marginTop: 0,
+      marginBottom: 0,
+    }),
+    option: (base, { isFocused, isSelected }) => ({
+      ...base,
+      color: '#344b63',
+      backgroundColor: isFocused ? '#e4e4e6' : isSelected ? '#f7f7fa' : '#fff',
+      '&:active': {
+        backgroundColor: '#e4e4e6'
+      },
+    }),
+  });
+
```

Comment:
It looks like we can still use `classNames` (https://deploy-preview-2289--react-select.netlify.com/upgrade-guide#new-styles-api) - not sure what our approach should be here though 😢